specification/communication/data-plane/Rooms/preview/2023-03-31-preview/communicationservicesrooms.json

```diff
@@ -0,0 +1,550 @@
+            "description": "Error response",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get participants": {
+            "$ref": "./examples/Rooms_GetParticipants.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Participants"
+        ],
+        "summary": "Upsert participants to a room",
+        "operationId": "Rooms_PatchParticipants",
+        "consumes": [
+          "application/merge-patch+json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "roomId",
+            "description": "Room id to patch participants.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "patchParticipantsRequest",
+            "description": "Participants to be patched to the room.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PatchParticipantsRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The participants were successfully added. The action returns an empty object.",
+            "schema": {
+              "type": "object"
+            }
+          },
+          "default": {
+            "description": "Error response",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Add participants": {
+            "$ref": "./examples/Rooms_AddParticipants.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CommunicationErrorResponse": {
+      "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+    },
+    "CreateRoomRequest": {
+      "description": "Request payload for creating new room.",
+      "type": "object",
+      "properties": {
+        "title": {
+          "description": "The room title.",
+          "type": "string",
+          "example": "Meeting with Alice and Bob"
+        },
+        "validFrom": {
+          "format": "date-time",
+          "description": "The timestamp from when the room is open for joining. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. If this value is not set, its value is set to a current date time by default.",
```

Comment:
nit: consider making description more succinct: 
"The timestamp from when the room is open for joining. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. The default value is the current date time."

Reply:
+1 ^

Reply:
resolved Minnie